db/re/item_db.txt

```diff
@@ -9832,7 +9832,7 @@
 18977,Old_Midas_Whisper,Old Midas Whisper,4,10,,100,,10,,1,0x00040000,56,2,256,,170,1,629,{ bonus bAllStats,1; .@r = getrefine(); bonus bBaseAtk,4*.@r; bonus bLongAtkRate,.@r; bonus bMaxHPrate,.@r/2; bonus bMaxSPrate,.@r/2; bonus2 bSkillAtk,"GN_CARTCANNON",15*.@r/2; },{},{}
 18978,Old_Magic_Stone_Hat,Old Magic Stone Hat,4,10,,100,,10,,1,0x00000200,56,2,256,,170,1,630,{ bonus bAllStats,1; .@r = getrefine(); bonus bMdef,5; bonus bMatkRate,.@r; bonus bMaxHPrate,.@r/2; bonus bMaxSPrate,.@r/2; bonus2 bSkillAtk,"WL_SOULEXPANSION",5*.@r/2; bonus2 bSkillCooldown,"WL_CRIMSONROCK",-1000-(.@r*100); },{},{}
 18979,Old_Blazing_Soul,Old Blazing Soul,4,10,,100,,10,,1,0x00008000,56,2,256,,170,1,631,{ bonus bAllStats,1; .@r = getrefine(); bonus bBaseAtk,4*.@r; bonus bCritical,2*.@r; bonus bMaxHPrate,.@r/2; bonus bMaxSPrate,.@r/2; bonus2 bSkillAtk,"SR_RAMPAGEBLASTER",15*.@r/2; bonus2 bSkillAtk,"SR_RIDEINLIGHTNING",20*.@r/2; },{},{}
-18980,Old_Wind_Whisper,Old Wind Whisper,4,10,,100,,10,,1,0x10000,56,2,256,,170,1,633,{ bonus bAllStats,1; .@r = getrefine(); bonus bMaxHPrate,.@r/2; bonus bMaxSPrate,.@r/2; bonus bMdef,5; bonus2 bSkillAtk,"SO_POISON_BUSTER",15*.@r/2; bonus2 bSkillAtk,"SO_VARETYR_SPEAR",20*.@r/2; },{},{}
+18980,Old_Wind_Whisper,Old Wind Whisper,4,10,,100,,10,,1,0x10000,56,2,256,,170,1,633,{ bonus bAllStats,1; .@r = getrefine(); bonus bAtk2,4*.@r; bonus bMatkRate,.@r; bonus bMaxHPrate,.@r/2; bonus bMaxSPrate,.@r/2; bonus bMdef,5; bonus2 bSkillAtk,"SO_POISON_BUSTER",15*.@r/2; bonus2 bSkillAtk,"SO_VARETYR_SPEAR",20*.@r/2; },{},{}
```

Comment:
Damage of SO_POISON_BUSTER and SO_VARETYR_SPEAR increase by 15% and 20% every 2 refines.
It should be
`bonus2 bSkillAtk,"SO_POISON_BUSTER",15*(.@r/2); bonus2 bSkillAtk,"SO_VARETYR_SPEAR",20*(.@r/2);`

Reply:
i have made the changes as requested, but i don't see the difference (?)

Reply:
It's the order of calculation. In BEDMAS, without the parentheses, the bonus gets calculated from left to right. For instance, the item with +3 refine will give `15 * 3 / 2 => 45 / 2 => 22%` bonus to Poison Buster. While Atemo's will give `15 * (3/2) => 15 * 1 => 15%`. There's no decimal points in rA's script engine so they get rounded down.